ansible/roles/monitoring-agent/files/collectd-plugins/marathon

```diff
@@ -1 +1 @@
-Subproject commit 11a7277c12174e02e68c2cc322b33768515d88dd
+Subproject commit 96fe35d3bf36f4bc5200f60f71d04de95e090f6c
```

Comment:
I don't think that's right.  Seems like you're reverting the marathon plugin to the previous, non working version.